src/legacy/core_plugins/interpreter/common/types/image.ts

```diff
@@ -17,10 +17,15 @@
  * under the License.
  */
 
-export const image = () => ({
-  name: 'image',
+import { ExpressionType } from '../../types';
+import { Render } from './render';
+
+const name = 'image';
+
+export const image = (): ExpressionType<typeof name, undefined> => ({
```

Comment:
nit: do you mean `undefined` or do you mean `any`?  Because `undefined` would imply that no one should have any context at all passed in, whereas `any` would mean the context is inconsequential... I think this more style than anything, though.

Reply:
I wasn't thinking about style here so much as what would happen if we were to go back and add a proper `Image` interface in the future... If we keep this as `undefined`, then TS will blow up as soon as you start writing other methods that rely on this being anything other than `undefined`.

Whereas if we put `any`, it would be easier to forget to come back and put the proper parameter in place later, giving us a false sense of type safety.

Reply:
Actually, I'm realizing now this doesn't need to be `undefined` at all, since we have a type for Image already in the Canvas `image()` function. Will update that here too.